pkg/sentry/kernel/task_run.go

```diff
@@ -132,6 +133,7 @@ func (t *Task) doStop() {
 // +stateify savable
 type runApp struct{}
 
+// +noescape
```

Comment:
FWIW, I think this is easily confused with `go:noescape`. Perhaps name it `checkescape`?

Reply:
This is a good name. I'll change it.

Reply:
Done.